packages/flutter/lib/src/foundation/isolates.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:isolate';
+
+import 'package:meta/meta.dart';
+
+/// Signature for the callback passed to [compute].
+///
+/// {@macro flutter.foundation.compute.types}
+///
+/// Instances of [ComputeCallback] must be top-level functions, not closures or
+/// instance or static methods of classes.
+///
+/// {@macro flutter.foundation.compute.limitations}
```

Comment:
I'm not familiar with the `{@` syntax, but this line looks a little suspicious. I'd expect either `{@endmacro}` to close the macro started above (like `{@endtemplate` below), or some content to follow this line 😕

Reply:
As far as I can tell, `{@template name} ... {@endtemplate}` declares a macro named "name", and `{@macro name}` inserts the macro named "name".